spec/acceptance/receipts_managing_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe 'Receipts managing', type: :feature do
   let(:user) { create(:user) }
+  let(:other_user) { create(:other_user) }
```

Comment:
Here you should be able to use the same factory and just change `email`, like this:

`let(:other_user) { create(:user, email: 'otheruser@example.com') }`